### PR TITLE
Exposed default time to live API on CosmosContainerProperties

### PR DESCRIPTION
**Pull request by kushagraThapar**
Direct port of this PR: https://github.com/Azure/azure-sdk-for-java/pull/6651
Addresses this: #5445

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java [hunk 1]
```diff
@@ -196,6 +196,46 @@ public CosmosContainerProperties setConflictResolutionPolicy(ConflictResolutionP
         return this;
     }
 
+    /**
+     * Gets the collection's default time-to-live value.
+     *
+     * @return the default time-to-live value in seconds.
+     */
+    public Integer getDefaultTimeToLive() {
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: shouldn't we include the unit?

`getDefaultTimeToLiveInSeconds()` ?

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java [hunk 2]
```diff
@@ -196,6 +196,46 @@ public CosmosContainerProperties setConflictResolutionPolicy(ConflictResolutionP
...
+     * <p>
+     * nonzero positive integer - indicates the default time-to-live value for all documents within the collection. This value can be overridden
+     * by individual documents' time-to-live value.
+     * <p>
+     * -1 - indicates by default all documents within the collection never expire. This value can be overridden by individual documents'
+     * time-to-live value.
+     *
+     * @param timeToLive the default time-to-live value in seconds.
+     */
+    public void setDefaultTimeToLive(Integer timeToLive) {
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: ditto

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: We can if we want to, I am fine with it. 
Though the API has been called `getDefaultTimeToLive()` in both v2 and v3. 
Should we change it ?

Comment (moderakh) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: Let's decide what is the right thing to do and we do that for v4 as v4 is not released yet.

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: Agreed, @kirankumarkolli @mbhaskar @David-Noble-at-work  do you guys have any opinion on this ?
Mo and I are in agreement of changing it to `getDefaultTimeToLiveInSeconds()` and `setDefaultTimeToLiveInSeconds()`

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProperties.java]: Done.

**Pull request merged**